output-data/changelogs/versions/38925.json

```diff
@@ -0,0 +1,67 @@
+[
+    {
+        "Description": "* Urn now reveals you after 20s from initial pickup",
+        "Tags": [
+            "Other"
+        ]
+    },
+    {
+        "Description": "* Urn sprint changed from +3/5 for winning/losing team to +2/6",
+        "Tags": [
+            "Other"
+        ]
+    },
+    {
+        "Description": "* First flex slot is now granted after 3 Guardians die instead of 4",
+        "Tags": [
+            "Other"
+        ]
+    },
+    {
+        "Description": "* Base kill bounty reduced from 225 to 150 (still scales to 1400 over time)",
+        "Tags": [
+            "Other"
+        ]
+    },
+    {
+        "Description": "* Respawn time at 20 minutes reduced from 45s to 40s",
+        "Tags": [
+            "Other"
+        ]
+    },
+    {
+        "Description": "* T3 Golden Statues spawn time moved from 25 min to 30 min",
+        "Tags": [
+            "Other"
+        ]
+    },
+    {
+        "Description": "* Range to attack lane Guardians increased from 19m to 20m",
```

Comment:
we need to make considerations for neutrals/npcs as well. I'm happy to defer that to another PR though

Reply:
Yeah we will likely find more tags we can parse for, though before I forgot I did these just now.